### PR TITLE
Banner to core

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

**Comment by alycrys** [comment]
> 

I'll need to rebase it on top of this (that one has the spacing foundation changes)
I thought to do the move to core first, in case we want to release it, and add the spacing changes when they're ready.

File: packages/core/stories/banner/banner.doc.stories.mdx [hunk 1]
```diff
@@ -29,28 +28,28 @@ You should always position the Banner component in the main content area, direct
...
 </Canvas>
 
+If you need to share multiple notifications of the same state with your users, you should summarize them in a single Banner
+
+If you have multiple notifications of different states, you should stack the respective banners from top to bottom,
+in the following order: Information, Error, Warning, Success.
+
 ## Emphasized
 
 Banner can be emphasized by passing in `emphasize={true}`.
```
Comment (sion-docs) [inline packages/core/stories/banner/banner.doc.stories.mdx:50]: Can we add more detail about what happens to a Banner when it's emphasized?

**Review by sion-docs — CHANGES_REQUESTED** [review]

File: packages/core/stories/banner/banner.doc.stories.mdx (hunk 1 above)
Comment (alycrys) [reply packages/core/stories/banner/banner.doc.stories.mdx:50]: How about this?

**Review by libertymayc — DISMISSED** [review]

File: .changeset/quiet-squids-push.md
Comment (origami-z) [inline .changeset/quiet-squids-push.md:1]: It's probably worth to mention the need of `BannerContent` vs pure children

Comment (alycrys) [reply .changeset/quiet-squids-push.md:1]: I've added it to the docs

File: .changeset/quiet-squids-push.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+"@salt-ds/core": patch
+"@salt-ds/lab": patch
+---
+
+Remove `announcement` and `disableAnnouncer` props from Banner
```
Comment (libertymayc) [inline .changeset/quiet-squids-push.md:6]: Need to add the variants to this (unless planning to first merge another pr)

File: packages/core/stories/banner/banner.doc.stories.mdx [hunk 3]
```diff
@@ -17,40 +16,36 @@ import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
...
+If you have multiple notifications of different states, you should stack the respective banners from top to bottom, in the following order: Information, Error, Warning, Success.
+
 ## Emphasized
 
-Banner can be emphasized by passing in `emphasize={true}`.
+You can make the banner more prominent by passing `emphasize={true}` which will apply a background to the component, as seen in the examples below:
 
 <Canvas>
-  <Story id="lab-banner--emphasized" />
+  <Story id="core-banner--emphasized" />
```
Comment (libertymayc) [inline packages/core/stories/banner/banner.doc.stories.mdx:48]:
```suggestion
  <Story id="core-banner--statuses-secondary" />
```

File: packages/core/stories/banner/banner.doc.stories.mdx [hunk 4]
```diff
@@ -17,40 +16,36 @@ import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
...
 <Canvas>
-  <Story id="lab-banner--statuses" />
+  <Story id="core-banner--statuses" />
 </Canvas>
 
+If you need to share multiple notifications of the same state with your users, you should summarize them in a single Banner
+
+If you have multiple notifications of different states, you should stack the respective banners from top to bottom, in the following order: Information, Error, Warning, Success.
+
 ## Emphasized
```
Comment (libertymayc) [inline packages/core/stories/banner/banner.doc.stories.mdx:43]: needs changing

**Comment by alycrys** [comment]
closing in favour of https://github.com/jpmorganchase/salt-ds/pull/1710

**Pull request closed without merging**